GEON-PPANG-iOS/Presentation/LogIn/ViewController/LogInViewController.swift

```diff
@@ -219,8 +219,10 @@ extension LogInViewController {
             switch status {
             case 200...204:
                 AnalyticManager.log(event: .general(.loginApp(loginType: AnalyticEventType.EMAIL.rawValue)))
-                let sceneDelegate = UIApplication.shared.connectedScenes.first?.delegate as? SceneDelegate
-                sceneDelegate?.changeRootViewControllerToTabBarController()
+                DispatchQueue.main.async {
+                    let sceneDelegate = UIApplication.shared.connectedScenes.first?.delegate as? SceneDelegate
+                    sceneDelegate?.changeRootViewControllerToTabBarController()
```

Comment:
p2:
이거 제가 Scenedelegate Utils 에 선언해둬서 아래처럼 바꾸기 가능
```suggestion
                    Utils.sceneDelegate?.changeRootViewControllerToTabBarController()
```

Reply:
4f5b1aff84fa1dc701f6363a853291ea19b06ce4